Externals.cfg

```diff
@@ -45,17 +45,18 @@ required = True
 protocol = git
 repo_url = https://github.com/NOAA-GSL/GSI.git
 # Specify either a branch name or a hash but not both.
-#branch = feature/rrfs
-hash = 32e9d20
+branch = feature/cmake-refactor
```

Comment:
This needs to be a hash. We will keep change the GSI code and hash is safer.

Reply:
Fixed.